.gitignore

```diff
@@ -22,3 +22,4 @@ _archive/
 _public/
 
 # Additional entries
+jsonnetfile.json
```

Comment:
I don't think this should be gitignored, as there's a possibility that we'll want to specify Jsonnet dependencies in this component at some point.

Reply:
Fair enough. It just got annoying to not be able to add just `git add .`

Reply:
That's what `.git/info/exclude` or a global `gitignore` is for ;)